### DISCUSSION TITLE
可以用MaaDebugger测试自定义任务吗？

### DISCUSSION THREAD

**Discussion by Oarwind** [MaaXYZ/MaaFramework#498] [General]
我注意到给出的样例都是启动MaaPiCli的，请问如果要在注册任务后使用MaaDebugger的话，应该怎么做呢？

**Comment by dongwlin** [comment]
请问，你的“注册任务”具体指什么？是指custom recognition/action，还是单纯指pipeline那些json？

**Reply by Oarwind** [reply]
是自定义识别/操作

**Comment by MistEO** [comment]
暂时不支持，有计划，但还没写 https://github.com/MaaXYZ/MaaFramework/issues/333

**Comment by weinibuliu** [comment]
https://github.com/MaaXYZ/MaaDebugger/commit/b0ef80d02cfa5d6ab436a80e37db8c984262a620 已支持调试 agent

**Comment by weinibuliu** [comment]
除 agent 外，正在计划支持 register 接口 https://github.com/MaaXYZ/MaaDebugger/issues/61
